implementations/elixir/ockam/ockam_services/lib/services/api.ex

```diff
@@ -92,20 +94,22 @@ defmodule Ockam.Services.API do
   def handle_request(module, request, state) do
     case module.handle_request(request, state) do
       {:reply, status, body, state} ->
-        reply = reply(request, status, body, state.address)
+        reply = reply(request, status, body, state.address, module)
         {reply, state}
 
       {:noreply, state} ->
+        ## No response, but still emit reply=false stop event
+        emit_request_stop(module, request, :noreply, state.address)
```

Comment:
Do we use this `{:noreply, state}` return value for the API worker, other than in the async API handlers?

I'm wondering if we could simply not emit the stop event here? Or differentiate between situations where we want to do it and those where we don't

Reply:
I think it still makes sense to report how long does it take to handle the request outside of async ops. We can differentiate these durations by `reply=true/false` tag.